core/tauri-runtime-wry/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = { workspace = true }
 rust-version = { workspace = true }
 
 [dependencies]
-wry = { version = "0.35.2", default-features = false, features = [ "file-drop", "protocol", "os-webview" ] }
+wry = { version = "0.36", default-features = false, features = [ "file-drop", "protocol", "os-webview" ] }
 tao = { version = "0.25", default-features = false, features = [ "rwh_05", "rwh_06" ] }
```

Comment:
I belive we don't need `rwh_05` anymore and should update `raw-window-handle` across the codebase to `0.6`